docs/clients/grpc/projections/README.md

```diff
@@ -0,0 +1,665 @@
+﻿# Projection management
+
+The various gRPC client APIs includes dedicated clients that allow you to manage projections.
+
+:::note
+Currently not all clients fully expose all operations.
+:::
+
+For a detailed explanation of projections, see the [server](/server/v21.2/docs/projections/) documentation.
+
+The full sample code shown here can be found on the respective [clients repositories](https://github.com/eventStore/?q=client)
+
+## Required packages
+<xode-group>
+<xode-block title="C#">
+
+```bash
+dotnet add package EventStore.Client.Grpc.ProjectionManagement --version 21.*
+```
+</xode-block>
+<xode-block title="Java">
+
+```
+# Maven
+<dependency>
+  <groupId>com.eventstore</groupId>
+  <artifactId>db-client-java</artifactId>
+  <version>1.0.0</version>
+</dependency>
+
+# Gradle
+implementation 'com.eventstore:db-client-java:1.0.0'
+
+# SBT
+libraryDependencies += "com.eventstore" % "db-client-java" % "1.0.0"
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```
+# Yarn
+$ yarn add @eventstore/db-client
+
+# NPM
+$ npm install --save @eventstore/db-client
+```
+</xode-block>
+<xode-block title="Rust">
+
+```
+No additional configuration is needed having Rust installed. Go check https://rustup.rs.
+```
+</xode-block>
+</xode-group>
+
+## Creating a client 
+
+Projection management operations are exposed through a dedicated client. 
+
+<xode-group>
+<xode-block title="C#" code="connectionString">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#createClient
+
+</xode-block>
+<xode-block title="Java" code="connectionString">
+
+<<< @/docs/clients/java/generated/1.0.0/samples/quick_start/QuickStart.java#createClient
+
+</xode-block>
+<xode-block title="NodeJS" code="connectionString">
+
+<<< @/docs/clients/node/generated/2.0.0/samples/get-started.js#createClient
+
+</xode-block>
+<xode-block title="Rust" code="connectionString">
+
+<<< @/docs/clients/rust/generated/1.0.0/samples/quickstart.rust#createClient
+
+</xode-block>
+</xode-group>
+
+## Restart the subsystem
+Restarts the entire projection subsystem. The user must be in the `$ops` or `$admin` group to perform this operation
```

Comment:
**SUGGESTION:** It'd be good to explain briefly what's the projection subsystem and why would you like to restart it.

Reply:
that's in the server doc if I'm not mistaken

Reply:
We should give at least basic information, so people don't need to jump back and forth to the docs. It may be a single sentence, but giving context, what are the consequences, etc. The same applies to other methods.

Reply:
> Mistaken , it's not there .
> Dunno why it needs restarting .